docs/Engineering/Services/data-engineering.md

```diff
@@ -0,0 +1,22 @@
+# Data Engineering
+
+Data Engineering is the part of a client's journey where they have lots of data but are not sure how to use it to make informed decisions. That's where we can help! We work with clients to understand their data needs and identify relevant sources of data. We can design and implement data pipelines and advise on different data-store technologies, such as databases, data lakes, data warehouses, ETL tools, and big data platforms.
+
+We understand the different stages involved in defining a Data Pipeline.
+
```

Comment:
The below sections need prefacing with "Let's dig into what makes up the Data Engineering service" covering off that each of these things can exist on their own or wrapped up together.  This is where we can get in pictures or give real-world examples.